### ISSUE TITLE
[react-bootstrap] ListGroupItem is not a constructor for JSX

### ISSUE THREAD

**Issue by thoni56** [DefinitelyTyped/DefinitelyTyped#33185]
If you know how to fix the issue, make a pull request instead.

- [x] I tried using the `@types/xxxx` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @....

@walkerburgin @vsiao @danilojrr @Batbold-Gansukh @octatone @chengsieuly @mretolaza @katbusch @vitosamson @LKay @aaronbeall @jrakotoharisoa @r3nya @t49tran @bes

```
    "@types/react": "^0.14.57",
    "@types/react-bootstrap": "^0.32.17",
    "bootstrap": "^4.3.1",
    "react": "^16.8.2",
    "react-bootstrap": "^1.0.0-beta.5",
```

**Comment by aaronbeall** [comment]
Those are the wrong types. As of `1.0.0` you should use the built-in types, not `@types/react-bootstrap`.

**Comment by thoni56** [comment]
Wouldn't it be nice if obsolete @types was removed from this and npm's repo since it is almost impossible to find documentation about what typings are available automatically when a package is installed. That would prevent you from going wrong when you follow some instructions in an article just half a year old.

There is probably a host of other similar issues that could be closed also. E.g. all 28 open issues tagged `react-bootstrap`.

**Comment by bes** [comment]
@thoni56 The typings here are very much relevant, since they are typings for react-bootstrap prior to v1.0.0, which targets Bootstrap v3. react-bootstrap > v1.0.0 targets Bootstrap v4 and has built in typings. Not everyone is interested in, or has the capacity to upgrade to Bootstrap v4 / react-bootstrap v1.0.0.

TL;DR these typings are not obsolete and should not be removed.

**Comment by thoni56** [comment]
Ok, understood. Is there someway to signal this so others don't have to come here, create an issue, and have this conversation again?

Version checks somewhere maybe? I'm fairly new to the storming world of Javascript, so I'm not sure what's possible. But one of the problems in Javascript land is the fast moving targets...

**Comment by aaronbeall** [comment]
@thoni56 Yeah, I think the issue here is that `@latest` of `react-bootstrap` and `@types/react-bootstrap` are not version aligned as of `1.0.0-beta`. I believe the solution is to [publish a stub definition](https://github.com/DefinitelyTyped/DefinitelyTyped#removing-a-package) for `@types/react-bootstrap`. cc @bes
